deployment/load_production_backup.sh

```diff
@@ -9,7 +9,7 @@ cd $(dirname $0)/.. # change to root directory
 USERNAME="evap"
 ENVDIR="/opt/evap/env"
 ADDITIONAL_ARGUMENTS=""
-[[ ! -z "$EVAP_RUNNING_INSIDE_TRAVIS" ]] && echo "Detected travis" && USERNAME="travis" && ENVDIR=~/virtualenv/python3.7 && ADDITIONAL_ARGUMENTS=" --noinput"
+[[ ! -z "$GITHUB_WORKFLOW" ]] && echo "Detected GitHub" && USERNAME="github" && ENVDIR=/usr/local && ADDITIONAL_ARGUMENTS=" --noinput"
```

Comment:
i don't know which `USERNAME` to put here and in `update_production.sh`

Reply:
when not specifying the user (by removing all the `sudo -H -u $USERNAME` parts) everything runs nicely, see https://github.com/janno42/EvaP/runs/1816468720